src/store/slices/node/actionsAsync.ts

```diff
@@ -559,113 +559,202 @@ const getPrometheusMetricsThunk = createAsyncThunk(
 );
 
 export const createExtraReducers = (builder: ActionReducerMapBuilder<typeof initialState>) => {
+  builder.addCase(getInfoThunk.pending, (state, action) => {
+    state.info.isFetching = true;
+  });
   builder.addCase(getInfoThunk.fulfilled, (state, action) => {
     if (action.payload) {
-      state.info = action.payload;
+      state.info.data = action.payload;
     }
+    state.info.isFetching = false;
+  });
+  builder.addCase(getInfoThunk.rejected, (state, action) => {
+    state.info.isFetching = false;
+  });
+  builder.addCase(getAddressesThunk.pending, (state, action) => {
+    state.addresses.isFetching = true;
   });
   builder.addCase(getAddressesThunk.fulfilled, (state, action) => {
     if (action.payload) {
-      state.addresses = action.payload;
+      state.addresses.data = action.payload;
     }
+    state.addresses.isFetching = false;
+  });
+  builder.addCase(getAddressesThunk.rejected, (state, action) => {
+    state.addresses.isFetching = false;
+  });
+  builder.addCase(getAliasesThunk.pending, (state, action) => {
+    state.aliases.isFetching = true;
   });
   builder.addCase(getAliasesThunk.fulfilled, (state, action) => {
     if (action.payload) {
-      state.aliases = action.payload;
+      state.aliases.data = action.payload;
     }
+    state.aliases.isFetching = false;
+  });
+  builder.addCase(getAliasesThunk.rejected, (state, action) => {
+    state.aliases.isFetching = false;
   });
   builder.addCase(getBalancesThunk.pending, (state, action) => {
     if (action.payload) {
-      state.balances.reloading = true;
+      state.balances.isFetching = true;
     }
```

Comment:
should this only change to true if there is a payload? what is the payload in this case?

Reply:
Good catch, this should change without the payload.